src/ast/ddl.rs

```diff
@@ -377,6 +377,7 @@ pub enum ColumnOption {
     DialectSpecific(Vec<Token>),
     CharacterSet(ObjectName),
     Comment(String),
+    Unsigned,
```

Comment:
I think `UNSIGNED` is a DataType (not an option on the column definition)

Specifically you can do something like 

```sql
mysql> select cast(x as unsigned) from bar;
+---------------------+
| cast(x as unsigned) |
+---------------------+
|                   1 |
|                NULL |
+---------------------+
2 rows in set (0.00 sec)
```

Reply:
Thanks for good review!
I think too, close this pull request, and I will retry.